cloud/scope/powervs_machine.go

```diff
@@ -178,9 +179,10 @@ func NewPowerVSMachineScope(params PowerVSMachineScopeParams) (scope *PowerVSMac
 		serviceInstanceName = fmt.Sprintf("%s-%s", params.IBMPowerVSCluster.GetName(), "serviceInstance")
 		if params.IBMPowerVSCluster.Spec.ServiceInstance != nil && params.IBMPowerVSCluster.Spec.ServiceInstance.Name != nil {
 			serviceInstanceName = *params.IBMPowerVSCluster.Spec.ServiceInstance.Name
+			zone = params.IBMPowerVSCluster.Spec.Zone
```

Comment:
will it make any difference? why don't we just pass params.IBMPowerVSCluster.Spec.Zone directly in the below function?

Reply:
Agree. changed.